### PR TITLE
Last active df per project

### PR DESCRIPTION
**Pull request by Omastto1**
is depended by ForloopAI/forloop_platform#802

### CODE REVIEW

File: forloop_modules/queries/db_model_templates.py [hunk 1]
```diff
@@ -256,8 +256,17 @@ class APIScanWebpage(BaseModel):
 #     dataset_uid: str
 #     sheet_name: str
 #     email: str
-    
-    
+
+
+class APILastActiveDFNodeUid(BaseModel):
```
Comment (DovaX) [inline forloop_modules/queries/db_model_templates.py:261]: Df instead of DF?

File: forloop_modules/queries/db_model_templates.py [hunk 2]
```diff
@@ -256,8 +256,17 @@ class APIScanWebpage(BaseModel):
...
+    project_uid: str = "0"
+    last_active_dataframe_node_uid: Optional[str]
+
+
+class APILastActiveDataFrame(BaseModel):
+    project_uid:str = "0"
+    page: int = 1
+    per_page: int = 100
```
Comment (DovaX) [inline forloop_modules/queries/db_model_templates.py:269]: records_per_page

File: forloop_modules/queries/node_context_requests_backend.py [hunk 3]
```diff
@@ -484,7 +484,8 @@ def delete_all_edges():
 
 def update_last_active_dataframe_node_uid(last_active_dataframe_node_uid: Optional[str]):
     payload = {
-        "last_active_dataframe_node_uid": last_active_dataframe_node_uid,
+        "project_uid": "1",
```
Comment (DovaX) [inline forloop_modules/queries/node_context_requests_backend.py:487]: why fixed 1, can we generalize it?

File: forloop_modules/queries/node_context_requests_backend.py [hunk 4]
```diff
@@ -498,7 +499,7 @@ def update_last_active_dataframe_node_uid(last_active_dataframe_node_uid: Option
 
 
 def get_last_active_dataframe_node_uid():
-    url = f'{BASE_API}/last_active_dataframe_node_uid'
+    url = f'{BASE_API}/last_active_dataframe_node_uid?project_uid=1'
```
Comment (DovaX) [inline forloop_modules/queries/node_context_requests_backend.py:502]: why fixed 1, can we generalize it?

**Review by DovaX — APPROVED** [review]

File: forloop_modules/queries/db_model_templates.py (hunk 1 above)
Comment (Omastto1) [reply forloop_modules/queries/db_model_templates.py:261]: pandas.DataFrame

File: forloop_modules/queries/db_model_templates.py (hunk 2 above)
Comment (Omastto1) [reply forloop_modules/queries/db_model_templates.py:269]: this should not have been here lol
will rename for next pr

**Review by DovaX — APPROVED** [review]

**Pull request merged**